Translation to Spanish [#1] 594 Words

Santu41 opened the pull request:
Cerré el antiguo pull request para poder poner las dos columnas juntas dentro de uno. Ambos están traducidos completamente al español y esperando corrección! saludos.. :100:

Santu41 committed:
Translation to Spanish [300 Worlds] mixins.md

Aquí dejo mi colaboración en el proyecto, la columna que me ha sido asignada esta traducida completamente al español.

Santu41 committed:
Update mixins.md

Santu41 committed:
Translation to Spanish [264 Words] plugins.md

Aquí dejo mi segunda colaboración en el proyecto, la columna que me ha sido asignada esta traducida completamente al español.

miljan-aleksic committed:
revisión

miljan-aleksic commented:
De nuevo muchas gracias @Santu41. Se me pasó por alto mencionarte las [Pautas de Traducción](https://github.com/vuejs-es/vuejs.org/wiki/Pautas-de-Traducci%C3%B3n), las cuales entre todos vamos mejorando.

Me he tomado la libertad de aplicar mi revisión directamente en tu repositorio para el mixins.md y me gustaría que en base a ello y las pautas mencionadas hagas tu mismo la revisión de Plugins.md.

Santu41 committed:
Corrección

acá esta la corrección espero no tener errores y de ser asi ir mejorando con el tiempo, saludos!

Santu41 commented:
Post corregido, espero no tener errores y de ser así poder ir mejorando en el transcurso del proyecto! saludos

miljan-aleksic commented on src/v2/guide/plugins.md:
```diff
@@ -4,78 +4,79 @@ type: guide
+1. Agregar algunos métodos o propiedades globales. Por ejemplo [vue-custom-element](https://github.com/karol-f/vue-custom-element)
 
-2. Add one or more global assets: directives/filters/transitions etc. e.g. [vue-touch](https://github.com/vuejs/vue-touch)
+2. Agregar uno o más recursos globales: directivas / filtros / transiciones, etc. Por ejemplo [vue-touch](https://github.com/vuejs/vue-touch)
```
Creo que es mejor seguir las mismas pautas de espaciado del texto original. La frase debería quedar como `Agregar uno o más recursos globales: directivas/filtros/transiciones, etc. Por ejemplo [vue-touch](https://github.com/vuejs/vue-touch)`

miljan-aleksic commented on src/v2/guide/plugins.md:
```diff
@@ -4,78 +4,79 @@ type: guide
+5. Una biblioteca que proporcione una API propia, mientras que al mismo tiempo introduce una combinación de las anteriores. Por Ejemplo [vue-router](https://github.com/vuejs/vue-router)
+
+Un Plugin Vue.js debería exponer un método `instalar` (Método de instalación). Se llamará al método con el constructor `Vue` como primer argumento, junto con las posibles opciones:
 
```
Me parece que este espacio sobra.

miljan-aleksic commented on src/v2/guide/plugins.md:
```diff
@@ -4,78 +4,79 @@ type: guide
-5. A library that provides an API of its own, while at the same time injecting some combination of the above. e.g. [vue-router](https://github.com/vuejs/vue-router)
+5. Una biblioteca que proporcione una API propia, mientras que al mismo tiempo introduce una combinación de las anteriores. Por Ejemplo [vue-router](https://github.com/vuejs/vue-router)
+
+Un Plugin Vue.js debería exponer un método `instalar` (Método de instalación). Se llamará al método con el constructor `Vue` como primer argumento, junto con las posibles opciones:
```
En este caso `install` no lo podemos traducir ya que es literalmente el nombre que Vue espera que el método tenga. Me parece bien dejar la traducción entre paréntesis pero tan solo `(instalar)`.

miljan-aleksic commented on src/v2/guide/plugins.md:
```diff
@@ -4,78 +4,79 @@ type: guide
-MyPlugin.install = function (Vue, options) {
-  // 1. add global method or property
-  Vue.myGlobalMethod = function () {
+MyPlugin.install = función (opciones de Vue) {
```
Para los ejemplos de código tan solo traducimos los comentarios asegurando que los mismos sigan siendo ejecutables.

miljan-aleksic commented on src/v2/guide/plugins.md:
```diff
@@ -4,78 +4,79 @@ type: guide
   })
 
-  // 3. inject some component options
+  // 3. inyectar algunas opciones de componentes
```
Respetemos los plurales/singulares originales -  `inyectar algunas opciones de componente`

miljan-aleksic commented on src/v2/guide/plugins.md:
````diff
@@ -4,78 +4,79 @@ type: guide
+Use los Plugins llamando al método global `Vue.use ()`:
 
 ``` js
 // calls `MyPlugin.install(Vue)`
````
por traducir

miljan-aleksic commented on src/v2/guide/plugins.md:
````diff
@@ -4,78 +4,79 @@ type: guide
 ```
 
-You can optionally pass in some options:
+Puede si lo desea descartar algunas opciones:
````
Hay un malentendimiento aquí. La traducción correcta sería `Opcionalmente puede declarar algunas opciones:`

miljan-aleksic commented on src/v2/guide/plugins.md:
````diff
@@ -4,78 +4,79 @@ type: guide
 ```
 
-`Vue.use` automatically prevents you from using the same plugin more than once, so calling it multiple times on the same plugin will install the plugin only once.
+`Vue.use` automáticamente lo previene de usar el mismo Plugin más de una vez, por lo que, al llamar varias veces a un mismo Plugin, este se instalará solo una vez.
````
Sugerencia: `...por lo que, al usarlo varias veces con el mismo Plugin, este se instalará solo una vez.`

miljan-aleksic commented on src/v2/guide/plugins.md:
```diff
@@ -4,78 +4,79 @@ type: guide
+`Vue.use` automáticamente lo previene de usar el mismo Plugin más de una vez, por lo que, al llamar varias veces a un mismo Plugin, este se instalará solo una vez.
 
-Some plugins provided by Vue.js official plugins such as `vue-router` automatically calls `Vue.use()` if `Vue` is available as a global variable. However in a module environment such as CommonJS, you always need to call `Vue.use()` explicitly:
+Algunos Plugins proporcionados por “Vue.js official plugins” como `vue-router`, citan automáticamente a `Vue.use ()` si `Vue` está disponible como una variable global. Sin embargo, en un módulo como CommonJS, usted siempre debe citar a `Vue.use ()` explícitamente:
```
Emplea "llamar" o "ejecutar" envez de "citar". Creo que se entiende mejor.

miljan-aleksic commented on src/v2/guide/plugins.md:
````diff
@@ -4,78 +4,79 @@ type: guide
 
 ``` js
-// When using CommonJS via Browserify or Webpack
+// Cuando usa CommonJS via Browserify o Webpack
````
`Cuando use...`

miljan-aleksic commented on src/v2/guide/plugins.md:
````diff
@@ -4,78 +4,79 @@ type: guide
 ```
 
-Checkout [awesome-vue](https://github.com/vuejs/awesome-vue#components--libraries) for a huge collection of community-contributed plugins and libraries.
+Consulte [awesome-vue](https://github.com/vuejs/awesome-vue#components--libraries)para obtener una gran colección de plugins y librerías proporcionados por la comunidad.
````
Cuidado con los espacios, falta uno después del enlace.

miljan-aleksic reviewed (changes requested):
Gracias! Aquí va mi revisión :)

Santu41 committed:
Correccion 2

Santu41 commented:
Todo arreglado! no entendí muy bien esto:
Para los ejemplos de código tan solo traducimos los comentarios asegurando que los mismos sigan siendo ejecutables.
pero lo demás perfecto, espero que me asignen una nueva columna, saludos!

miljan-aleksic committed:
revisión

miljan-aleksic commented:
Me refería a que los codigo de ejemplo tienen que funcionar si se copian y usan, ser ejecutables. Si traducimos la logica ya no son ejecutables. Por lo que solo traducimos los comentarios.

Gracias por tu colaboración @Santu41. Te he asignado "Deploying for Production" :)
